BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudFile.cs

```diff
@@ -96,16 +108,23 @@ public BrainCloudFile(BrainCloudClient client)
             FailureCallback failure = null,
             object cbObject = null)
         {
+            byte[] byteData = GetByteData(fileData);
```

Comment:
No, fileData in parameter should be `byte[]`. We don't want to convert to string anymore! So you don't need this `GetByteData()` anymore here.

But actually, we want to deprecate the old call, so you might want to create an overloading method that still takes the string, and then add the deprecated tag.